common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -36,6 +36,12 @@ public class RssBaseConf extends RssConf {
       .defaultValue("GRPC")
       .withDescription("Shuffle server type, default is grpc");
 
+  public static final ConfigOption<Integer> SERVER_PORT_MAX_RETRIES = ConfigOptions
```

Comment:
What's the difference between `SERVER_PORT_MAX_RETRIES` and `RSS_RANDOM_PORT_ATTEMPT`?

Reply:
RSS_RANDOM_PORT_ATTEMPT is for check random port try, SERVER_PORT_MAX_RETRIES  is for start server failed try. whether it is necessary of the random port check? if isPortAvailable is not necessary ,the RSS_RANDOM_PORT_ATTEMPT conf can removed.

Reply:
Let's remove it.